Dockerfile

```diff
@@ -42,5 +44,6 @@ RUN RAILS_ENV=production \
   rm -rf /app/tmp/
 
 
+ENTRYPOINT [ "/app/bin/envconsul-wrapper" ]
 
-CMD ["./entrypoint.sh"]
+CMD ["/app/bin/startup"]
```

Comment:
I'm assuming `ENTRYPOINT` must be specified prior to every `CMD` or can you just define it once and all commands are run through it afterwards?

Also, why does it look like we're starting the app twice?

Reply:
`CMD` is passed to `ENTRYPOINT` as a positional argument, `ENTRYPOINT` is optional, but every Dockerfile needs a `CMD` and each stage needs a `CMD`  this is a multi-stage docker file, `base` and `production`